### PR TITLE
User can view their entries | Adrienne Yung

### PR DESCRIPTION
**Pull request by adrienneyung64**
code-journal-feature-2 submit2

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Adrienne, your header's background color should fill the entire viewport, not just the width of the container, please revise and resubmit with a new screenshot

**Comment by adrienneyung64** [comment]
code-journal-feature-2 submit3

File: css/styles.css [hunk 1]
```diff
@@ -36,21 +67,36 @@ h3 {
...
-.button {
+.save-button {
   text-align: right;
   margin-top: 30px;
 }
 
+.new-button {
+  margin-top: 30px;
+  margin-bottom: 30px;
+  margin-left: 595px;
```
Comment (codingcodymiller) [inline css/styles.css:88]: This isn't the best way to handle this, I would make use of flexbox here, look into the `space-between` value for `justify-content`

File: js/main.js [hunk 2]
```diff
@@ -14,18 +14,80 @@ function changePhoto(event) {
...
+  divThree.appendChild(divFive);
+  var entryParagraph = document.createElement('p');
+  entryParagraph.textContent = entry.notes;
+  divFive.appendChild(entryParagraph);
+  return listEntry1;
+}
+
+window.addEventListener('DOMContentLoaded', appendToUl);
+var unorderedList = document.querySelector('ul');
+function appendToUl(event) {
```
Comment (codingcodymiller) [inline js/main.js:66]: You should revise the name of this function, now that it also changes the view. Maybe something like `handlePageLoad`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by adrienneyung64** [comment]
code-journal-feature-2 submit4

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**